src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/GenericDefinitionEETypeNode.cs

```diff
@@ -30,6 +31,11 @@ public override bool ShouldSkipEmittingObjectNode(NodeFactory factory)
             return false;
         }
 
+        protected override DependencyList ComputeNonRelocationBasedDependencies(NodeFactory factory)
```

Comment:
I don't particularly like the factoring of the EEType nodes here. The base class for all EETypes represents an unconstructed EEType instead of representing what is the common functionality between all EEType nodes. This necessitates suppressing various behaviors of the base class. I would like to spend some time refactoring this into a shape that makes more sense.